test/live_test.sh

```diff
@@ -211,9 +233,15 @@ cleanup_booth() {
 cleanup_dep_rsc() {
 	local dep_rsc=`get_rsc`
 	test -z "$dep_rsc" && return
-	local h procs
+	local cmd h procs
+	case "$cli_driver" in
+	native) cmd="crm_resource --cleanup --resource \"$dep_rsc\"; crm_resource --wait";;
```

Comment:
Why is here "crm_resource --wait" in the extra invocation? There may also be a race condition if the PE doesn't change the state before the second command is run.